python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -362,35 +366,56 @@ def makePlot(
             plotInfo["plotName"] = self.plotName
 
         # Plotting customization.
+        colorbarLocation = "right"
         colorbarTickLabelSize = 14
         colorBarAspect = 16
+        colorBarFraction = 0.07  # w.r.t. the axes size.
+        colorBarPad = 0  # w.r.t. the axes size.
         rcparams = {
-            "axes.labelsize": 18,
+            "axes.labelsize": 22 if plotConfig.publicationStyle else 19,
             "axes.linewidth": 1.8,
-            "xtick.labelsize": 13,
-            "ytick.labelsize": 13,
+            "xtick.labelsize": 15 if plotConfig.publicationStyle else 14,
+            "ytick.labelsize": 15 if plotConfig.publicationStyle else 14,
         }
-        zoomFactors = plotConfig.zoomFactors
 
-        # Muted green for the full map, and muted red and blue for the two
-        # zoomed-in maps.
-        histColors = ["#265D40", "#8B0000", "#00008B"]
         if plotConfig.publicationStyle:
-            histColors = ["#440154", "#31688e", "#35b779"]
+            zoomFactors = []
+            _LOG.info(
+                "Zoom factors are not used in publication-style plots. "
+                "Only the full-tract map will be plotted."
+            )
+            histColors = ["#440154"]
+            rasterized = False
+            labelpad = 14
+        else:
+            zoomFactors = plotConfig.zoomFactors
+            # Muted green for the full map, and muted red and blue for the two
+            # zoomed-in maps.
+            histColors = ["#265D40", "#8B0000", "#00008B"]
+            rasterized = True  # Plot with rasterized graphics.
+            labelpad = 11
 
         toolName = data["data"].ref.datasetType.name
         mapName = toolName.replace("_map_", "_")
         mapData = data["data"].get()
 
         with rc_context(rcparams):
-            fig = make_figure(figsize=(16, 16))
-            ax1 = fig.add_subplot(221)
-            ax2 = fig.add_subplot(222)
-            ax3 = fig.add_subplot(223)
-            ax4 = fig.add_subplot(224)
+            if plotConfig.publicationStyle:
+                # Make a single plot for the full tract.
+                fig = make_figure(figsize=(8, 8))
```

Comment:
The figsize is set in the style file, does it not work for this plot?

Reply:
Everything's tuned for an 8x8 figure. If you switch sizes, you'll need to adjust paddings, font sizes, and other things because arbitrary figsize changes without these tweaks can lead to unwanted results.